docs/documentation/upgrading/topics/migrate_db.adoc

```diff
@@ -10,6 +10,11 @@ database is automatically migrated when you start the new installation for the f
 Before you migrate the database, shut down all {project_name} nodes running the old version of {project_name}.
 ====
 
+[NOTE]
+====
+Migration is not supported with the H2 based `dev-file` database type.
```

Comment:
```suggestion
Migration is not supported with the H2 based `dev-file`(persistent) database type.
```

Reply:
I believe it's implied it's  persistent when it's `dev-file` and not `dev-mem`.